src/main/java/TaskList.java

```diff
@@ -0,0 +1,207 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TaskList {
+
+    public TaskList() {}
+
+    /**
```

Comment:
For all of your JavaDocs, 
1. There should be punctuations at the end of every sentence in your description and every parameter definition. (Note that the descriptions are not supposed to be too long unless it is completely necessary)
2. There should be an empty line separating the description and parameter fields.
3. All sentences should have proper capitalisation.
4. JavaDocs of a method should be directly above a method, without a line separating them.

Below is a screenshot from the Java Coding Standard page in the "comments" section for your reference. 

![image](https://user-images.githubusercontent.com/60812414/106601349-83764c80-6596-11eb-81a0-49178c932968.png)